### PR TITLE
AAC-168 Notification Banner Helper and Spec Tests

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
#### What
Introduces a notification banner helper and spec tests

#### Ticket

[AAC-168 Add Ruby helper for notification banners including a failure option](https://dsdmoj.atlassian.net/browse/AAC-168)

### CODE REVIEW

File: app/helpers/govuk_design_system_helper.rb [hunk 1]
```diff
@@ -46,8 +46,35 @@ def govuk_summary_list_entry(key, value, tag_options_key = {}, tag_options_value
     end
   end
 
+  def govuk_notification_banner(text, key = 'Important', tag_options = {})
```
Comment (naseberry) [inline app/helpers/govuk_design_system_helper.rb:49]: Pattern: I would suggest setting `nil` as the default for the `text` arg

File: app/webpack/stylesheets/components/_notification_banner.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    color: govuk-shade($govuk-error-colour, 20%);
+  }
+
+  .govuk-notification-banner__link:active {
+    color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__link:focus {
+    color: $govuk-text-colour;
+  }
```
Comment (naseberry) [inline app/webpack/stylesheets/components/_notification_banner.scss:20]: I would recommend using the [govuk-link-style-error mixin](https://github.com/alphagov/govuk-frontend/blob/main/package/govuk/helpers/_links.scss#L130) to keep the style consistent.

File: app/helpers/govuk_design_system_helper.rb (hunk 1 above)
Comment (naseberry) [reply app/helpers/govuk_design_system_helper.rb:49]: I would consider allowing this method to accept a content block as we may wish to add more than just a paragraph.

See: https://design-system.service.gov.uk/components/notification-banner/success/index.html

Comment (MalcolmVonMoJ) [reply app/helpers/govuk_design_system_helper.rb]: Done and done

**Review by pking-moj — APPROVED** [review]

**Pull request merged**